content/enterprise_influxdb/v1.8/administration/renew-license.md

```diff
@@ -15,7 +15,8 @@ Use this procedure to renew or update an existing license key or file, switch fr
 
 To update a license key or file, do the following:
 
-1. **Add the license key or file** to your [meta nodes](/enterprise_influxdb/v1.8/administration/config-meta-nodes/#enterprise-license-settings) and [data nodes](/enterprise_influxdb/v1.8/administration/config-data-nodes/#enterprise-license-settings) configuration settings. For more information, see [how to configure InfluxDB Enterprise clusters](/enterprise_influxdb/v1.8/administration/configuration/).
-2. **On each meta node**, run `service influxdb-meta restart`, and wait for the meta node service to come back up successfully before restarting the next meta node.
+1. If you are switching from a license key to a license file (or vice versa), delete your existing license key or file.
```

Comment:
the license file is something sent to the customer from the support team (or CSMs).
In the configuration file, we specify the name of the license file and where it is located.

In many cases, the customer will likely receive a new file -- which may have the same name/or they may choose to rename it to match the name of the current file.  However, the contents of that file are clearly different.  So, if the customer just drops the new file on top of the one old...replacing its contents, it still needs to be "re-loaded" by the influx process.

Reply:
Thanks, @timhallinflux, my thought was--all use cases need the restart meta/reload data steps...but if they're switching from file to key or key to file, they'd need to delete the old one first?